lib/plugins/inspec-sign/lib/inspec-sign/base.rb

```diff
@@ -92,12 +92,16 @@ def self.profile_sign(profile_path, options)
         Inspec::UI.new.exit(:usage_error)
       end
 
-      def self.profile_verify(signed_profile_path)
+      def self.profile_verify(signed_profile_path, runner_call = false)
```

Comment:
How about "silent = false"? The flag seems to control whether the check outputs messages to the UI.